mysql_scripts/table_creation/FullDB.sql

```diff
@@ -73,9 +73,16 @@ CREATE TABLE Client (
 
 CREATE TABLE Appointment (
 	appointmentId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
-	scheduledTime DATETIME NOT NULL,
+    scheduledTime DATETIME NOT NULL,
 	createdAt DATETIME NOT NULL DEFAULT NOW(),
-	arrivedAt DATETIME NULL,
+    language VARCHAR(255) NOT NULL,
```

Comment:
I think the plan for language was to have it as a Question asked on the signup form, and then leaving it as a question. But I think maybe it should actually be on Client. Idk we should talk this one through--I'm torn on where to put it

Reply:
This is something I'm hoping to cover in standup tonight